modules/ROOT/pages/Development/Satisfactory/Audio.adoc

```diff
@@ -7,14 +7,219 @@ This page is a work in progress.
+
+Name the new asset **Exactly** the same name as the event in Wwise.
+
+[NOTE]
+====
+If the asset name is exactly the same as the Wwise event, Unreal Engine will automatically populate the correct event information to the variables below.
+====
+
+Verify that the event has the correct information by opening the event in Unreal.
+
+* Verify that the `Wwise Short ID` matches the event ID in Wwise.
+* Verify the `Wwise Name` matches the event name.
+
+image:Satisfactory/Wwise/Wwise_UEAkEvent_Verify.png[Wwise_UEAkEvent_Verify, 500]
+
+If everything has worked correctly, you can now test the event by right-clicking the event in Unreal and clicking `Play Event`
+
+=== Playing AKEvents In Blueprints
+
+==== Fire & Forget Events
+
+If you would like to play a one-off AK event, such as a spark sound, you will need to use the following method in your blueprint.
+
+image:Satisfactory/Wwise/Wwise_UEAkEvent_FireAndForget.png[Wwise_UEAkEvent_FireAndForget, 100%]
+
+This will first create a new `AK Component` at the blueprint's root component and then play the AK Event specified.
+
+Once the AK event has ended, you must clean up the `AK Component` using the callback event.
+
+==== Looping Events
+
+If you would like to play a sound effect that might loop, for example, a humming/buzzing sound for a building.
+
+The best method is to add an `AK Component` to your blueprint and set the `AK Audio Event` variable on the component.
+
+You must ensure that you stop the event when the actor is destroyed.
+
+image:Satisfactory/Wwise/Wwise_UEAkEvent_EndPlay.png[Wwise_UEAkEvent_EndPlay, 100%]
+
+
+=== Setting Up Attenuation
+
+If you plan to use Wwise audio for buildings, you will need to set up attenuation. By default, the audio will play even if you are on the other side of the map.
+
+Wwise has a great tutorial on how to set this up here: https://youtu.be/lOP0seni1DY[Wwise & Unreal | Adding 3D Audio With Attenuation]
+
+=== Updating Wwise Values From Your Mod
+
+Wwise uses a feature called Real-Time Parameter Control (RTPC) that allows game engines like Unreal or Unity to change values on the Wwise software in real-time.
+
+An example of this would be if, for example, you have a building and want the sound volume to fluctuate based on the recipe's progress.
+
+You can set up an RTPC in Wwise and Unreal and update the value in your building's blueprint.
+
+To set these up, in your Wwise project, go to Project Explorer and the `Game Syncs` tab.
+
+Under `Game Parameters -> Default Work Unit` create a new game parameter by right-clicking the `Default Work Unit` and then `New Child -> Game Parameter`
+
+image:Satisfactory/Wwise/Wwise_RTPC_Create.png[Wwise_RTPC_Create, 100%]
+
+Name this whatever you want.
+
+Open the newly created Game Parameter, and you should see the following in the main section.
+
+image:Satisfactory/Wwise/Wwise_RTPC_Range.png[Wwise_RTPC_Range, 100%]
+
+You will need to set the range. I recommend using 0 to 1.
```

Comment:
```suggestion
The parameter must have a range defined.
A useful range is `0.0` to `1.0` since many existing gameand engine systems already use that range as a start-finish, minimum-maximum, or off-on scale.
```